packages/core/core/src/TargetResolver.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+import type {FilePath, PackageJSON, Target, Environment} from '@parcel/types';
+import config from '@parcel/utils/config';
+import path from 'path';
+
+const DEFAULT_ENGINES = {
+  browsers: ['> 0.25%'],
+  node: '8'
+};
+
+const DEFAULT_ENV = {
+  context: 'browser',
+  includeNodeModules: true,
+  engines: DEFAULT_ENGINES
```

Comment:
Does it matter that the `DEFAULT_ENV.context` is 'browser', but `DEFAULT_ENGINES` includes a node version?

Reply:
Mmm, well the compiler should use the engine related to the context it is in so probably ok...